pkg/project/project.go

```diff
@@ -34,6 +36,60 @@ func Create(client *occlient.Client, projectName string) error {
 	return nil
 }
 
+// Delete deletes the project with name projectName and sets the project with name currentProject as current project and returns errors if any
+func Delete(client *occlient.Client, projectName string) error {
+	currentProject := GetCurrent(client)
+
+	projects, err := List(client)
+	if err != nil {
+		return errors.Wrapf(err, "unable to fetch list of projects")
+	}
+
+	//Iterate the project list and see the expected change post deletion
+	for ind, prj := range projects {
+		if prj.Name == projectName {
+			projects = append(projects[:ind], projects[ind+1:]...)
+		}
+	}
+
+	// If there will be any projects post the current deletion,
+	// Randomly choose a project from remainder of project list to set as current
```

Comment:
This is bad, we should pick the next available project in the list, not a random one.

Reply:
Hmm we are actually using the first project in list I'll rephrase the words to indicate the same